main.py

```diff
@@ -38,6 +39,7 @@
 
 def main(POST_ID=None):
     cleanup()
+    change_background_resolution()
```

Comment:
no need in that
+ functions with `global` are bad practice, try to avoid using them